VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -111,6 +121,11 @@ public void onPageLoaded(List<T> newItems, boolean hasMore) {
                     hasMoreItems = hasMore;
                     loading = false;
                 }
+
+                private boolean isResetCalled() {
+                    // Disregard result, since reset() was called
```

Comment:
This comment is not appropriate here, as this is just a getter method. Maybe it should be moved to the method documentation:

For example:

> Return whether this callback has been abandoned by the controller.

Or: 

> Return whether this callback has been abandoned because of the controller being reset.